app/controllers/resources_controller.rb

```diff
@@ -65,6 +66,7 @@ def update
     result = BackgroundJobResult.create(output: {})
     UpdateJob.perform_later(model_params: JSON.parse(cocina_dro.to_json), # Needs to be sidekiq friendly serialization
                             signed_ids: signed_ids(params),
+                            globus_ids: globus_ids(params),
```

Comment:
Is this the automagically provided `params` variable? If so, I don't think it needs to be passed to these methods as it should already be available. (I should have caught this in an earlier review, my bad!)